frontend/src/Constants.js

```diff
@@ -0,0 +1,9 @@
+const prod = {
+  WS_URL: "ws://example.com"
+};
+
+const dev = {
+  WS_URL: "ws://localhost:" + (process.env.SERVER_PORT || 8080)
```

Comment:
"localhost" should be changed to `window.location.host`

Reply:
Shouldn't this be `window.location.hostname`, since we just care about the hostname and not the port in this case?

Reply:
Another thing, `process.env.SERVER_PORT` is currently not recognized as CRA only defines environment variables starting with `REACT_APP_`.
Should we rename `SERVER_PORT` to `REACT_APP_SERVER_PORT` for both the backend and frontend? Is there a better solution to this?